cmd/kubeless/function.go

```diff
@@ -207,3 +208,17 @@ func getFunctionDescription(funcName, ns, handler, funcContent, deps, runtime, t
 	}
 	return
 }
+
+func getDeploymentStatus(cli kubernetes.Interface, funcName, ns string) (string, error) {
+	dpm, err := cli.ExtensionsV1beta1().Deployments(ns).Get(funcName, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+	status := fmt.Sprintf("%d/%d", dpm.Status.ReadyReplicas, dpm.Status.Replicas)
+	if dpm.Status.ReadyReplicas == dpm.Status.Replicas {
```

Comment:
we can improve on that later. There will be cases when we scale where some functions are ready and others not, but the service will be working.

Reply:
I can set it as ready if the ready replicas are `> 0`